grype/db/v6/description.go

```diff
@@ -0,0 +1,87 @@
+package v6
+
+import (
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/OneOfOne/xxhash"
+	"github.com/spf13/afero"
+
+	"github.com/anchore/grype/grype/db/internal/schemaver"
+	"github.com/anchore/grype/internal/file"
+)
+
+const DescriptionFileName = "metadata.json"
```

Comment:
It always throws me off a bit that this is sometimes called Description and sometimes called metadata. Is moving to schema v6 the right time to either call it `description.json` or call the struct `Metadata`? Not a blocker, but having two names has always confused me.

Reply:
I went with description.json